react-native-template-reason/src/App.re

```diff
@@ -4,55 +4,49 @@
  *
  * Converted from https://github.com/facebook/react-native/blob/d752446b23e0a12171ca0c01d20f5be625f40455/template/App.js
  */
-open BsReactNative;
+open ReactNative;
 
 let instructions =
-  switch (Platform.os()) {
-  | IOS(_) => "Press Cmd+R to reload,\n" ++ "Cmd+D or shake for dev menu"
-  | Android =>
+  switch (Platform.os) {
+  | os when os == Platform.ios =>
+    "Press Cmd+R to reload,\n" ++ "Cmd+D or shake for dev menu"
+  | os when os == Platform.android =>
     "Double tap R on your keyboard to reload,\n"
     ++ "Shake or press menu button for dev menu"
-  | exception (Platform.UnknownPlatform(p)) when p == "web" => "Press Cmd+R to reload."
-  | exception (Platform.UnknownPlatform(_p)) => ""
+  | os when os == Platform.web => "Press Cmd+R to reload."
```

Comment:
Won't this give a warning because the pattern matching might not be exhaustive?

You can add

```
_ => ""
```

at the end then.

Reply:
+1